util/crs-rules-check/rules-check.py

```diff
@@ -694,6 +720,9 @@ def msg(msg):
         try:
             with open(f, 'r') as fp:
                 fromlines = fp.readlines()
+                if f.startswith("crs-setup.conf.example"):
+                    fromlines = removecomments("".join(fromlines)).split("\n")
+                    fromlines = [l + "\n" for l in fromlines]
```

Comment:
Wouldn't this result in double newlines for each line? `remove_comments` already joins lines with `\n`.

Reply:
Thanks for the explanation.